fare/file_management/models.py

```diff
@@ -0,0 +1,20 @@
+"""MyRecord class model."""
+
+from invenio_records_files.api import Record
+from invenio_records_files.models import RecordsBuckets
+
+
+class MyRecord(Record):
+    """MyRecord class."""
+
+    def delete(self, force=True):
```

Comment:
Always remember to log the important actions, delete is definitely fundamental to log.